<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Station_add</class>
 <widget class="QDialog" name="Station_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Location</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBoxAddStation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStations">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>591</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditArrival">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrival</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Departure</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditDeparture">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCDT">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Direction</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditCDT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>118</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPASS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass non-stop</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSplit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Split Here</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxJoinOther">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Join Other</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSplitForward">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSplitReverse">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Other Service</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditSplit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEditSplitRef">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDeleteEntry">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxAddStation</sender>
   <signal>accepted()</signal>
   <receiver>Station_add</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxAddStation</sender>
   <signal>rejected()</signal>
   <receiver>Station_add</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
